interface/patient_file/front_payment.php

```diff
@@ -605,18 +586,8 @@ function toencounter(enc, datestr, topframe) {
 <script language="JavaScript">
     <?php require($GLOBALS['srcdir'] . "/restoreSession.php"); ?>
 function closeHow(e) {
-    if (top.tab_mode) {
-        top.activateTabByName('pat', true);
-        top.tabCloseByName(window.name);
-    } else {
-       if (opener) {
-            if (opener.name === "left_nav") {
-                dlgclose();
-                return;
-            }
-        }
-        window.history.back();
-    }
```

Comment:
TODO
May need to revert the above change since it will catch if being used as pop up.